lib/tests/streamlit/elements/select_slider_test.py

```diff
@@ -262,3 +264,69 @@ def test_label_visibility_wrong_value(self):
             "Unsupported label_visibility option 'wrong_value'. Valid values are "
             "'visible', 'hidden' or 'collapsed'.",
         )
+
+
+def test_select_slider_enum_coercion():
+    """Test E2E Enum Coercion on a select_slider."""
+    at = AppTest.from_string(
```

Comment:
@AnOctopus I actually thought `from_function` would be more difficult since it would probably make the type checker throw all sorts of errors. I am happy to change it out, though.